src/ui/FileViewer/CloseButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+export function CloseButton({ props }: {
```

Comment:
You are probably trying to do this ->  ~
`export function DeleteButton(props: FileViewerComponentProps): React.Element =>`

and in Fileviewer -> 
```
<DeleteButton  {... props} />
```
^^ this is little unoptimal, but is oaky :)

Reply:
unoptimal > You are passing values that are not all useful ~ but... Its okay :)

Reply:
Update -> If you use `React.ReactNode` instead of `React.Element ` it will support null, and you dont need to `</>`